activesupport/test/core_ext/duration_test.rb

```diff
@@ -54,11 +54,11 @@ def test_since_and_ago_with_fractional_days
   def test_since_and_ago_with_fractional_weeks
     t = Time.local(2000)
     # since
-    assert_in_delta((7 * 36).hours.since, 1.5.weeks.since, 1)
-    assert_in_delta((7 * 24 * 1.7).hours.since, 1.7.weeks.since, 1)
+    assert_equal((7 * 36).hours.since(t), 1.5.weeks.since(t))
+    assert_in_delta((7 * 24 * 1.7).hours.since(t), 1.7.weeks.since(t), 1)
     # ago
-    assert_in_delta((7 * 36).hours.ago, 1.5.weeks.ago, 1)
-    assert_in_delta((7 * 24 * 1.7).hours.ago, 1.7.weeks.ago, 1)
+    assert_equal((7 * 36).hours.ago(t), 1.5.weeks.ago(t))
+    assert_in_delta((7 * 24 * 1.7).hours.ago(t), 1.7.weeks.ago(t), 1)
```

Comment:
This was a backport from a upstream fix in activesupport. See https://github.com/Genius/rails2/commit/8f71612b48a341d2579b4196fd30a43de4e267f4#diff-5711b20159cce378efc8716683a03b38 for more info.